internal/semaphore_test.go

```diff
@@ -11,8 +11,8 @@ import (
 	"sync"
 	"testing"
 
-	. "go.mongodb.org/mongo-driver/internal"
 	"github.com/stretchr/testify/require"
+	. "go.mongodb.org/mongo-driver/internal"
```

Comment:
No need to change. But is this reordering from formatting/linting?

Reply:
yeah, this is from gofmt